woocommerce/woocommerce-ios
# Add tracks for Customer Search

Closes: #7741

### Description
This PR:
1. Adds tracking to the Customer Search project.
2. Removes the feature flag.

The Android tracks counterpart uses the same names:

- `order_creation_customer_search`: Triggered each time a new query is performed.
- `order_creation_customer_added`: Triggered when a Customer is selected. 

### Testing instructions
1. Go to Orders > + > Create new Order > + Add Customer details > Tap on the magnifying glass icon:
3. Search for a Customer name
4. Confirm that once the search completes, the following is logged in the console:
```
🔵 Tracked order_creation_customer_search, properties: [AnyHashable("is_wpcom_store"): false, AnyHashable("blog_id"): 121710041]
```
5. Tap on it:
6. Confirm that once tapped, the following is logged in the console: 
```
🔵 Tracked order_creation_customer_added, properties: [AnyHashable("is_wpcom_store"): false, AnyHashable("blog_id"): 121710041]
```

- [x] The PR must be assigned the Tracks label. Not applicable.
- [x] The tracks events must be validated in the Tracks system.
<img width="1298" alt="Screenshot 2022-11-01 at 13 31 52" src="https://user-images.githubusercontent.com/3812076/199159158-742e8a18-ebd4-4a03-b2a9-ecf2cd6ab863.png">

- [x] Verify the internal tracks spreadsheet has also been updated.
